app/src/main/res/layout/fragment_stocks.xml

```diff
@@ -49,22 +49,22 @@
                 android:layout_marginStart="@dimen/margin_padding_size_small"
                 android:textSize="@dimen/text_size_smaller"
                 app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintStart_toEndOf="@+id/textview_market_string"
-                app:layout_constraintTop_toBottomOf="@+id/spinner_stocks" />
+                app:layout_constraintStart_toEndOf="@id/textview_market_string"
+                app:layout_constraintTop_toBottomOf="@id/spinner_stocks" />
         </androidx.constraintlayout.widget.ConstraintLayout>
 
         <ImageView
             android:id="@+id/imageview_conversion_arrows"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:contentDescription="@string/conversion_icon_description"
-            app:layout_constraintEnd_toStartOf="@+id/spinnerCurrency"
-            app:layout_constraintStart_toEndOf="@+id/constraintLayout"
+            app:layout_constraintEnd_toStartOf="@id/spinner_currency"
+            app:layout_constraintStart_toEndOf="@id/constraintLayout"
             app:layout_constraintTop_toTopOf="parent"
             app:srcCompat="@drawable/ic_arrows" />
 
         <Spinner
-            android:id="@+id/spinnerCurrency"
+            android:id="@+id/spinner_currency"
             android:layout_width="128dp"
```

Comment:
All layout constants should be in `res/values` folder

Reply:
You mean `128dp` in this case?